documentation/library-reference/source/system/file-system.rst

```diff
@@ -885,19 +952,22 @@ File-System module.
      instantiated, this method calls the generic function
      :gf:`type-for-file-stream`.
 
-     The *filename* init-keyword should be a string naming a file. If
+     The *locator* init-keyword should be a string naming a file. If
      the :doc:`Locators <../system/locators>` library is in use,
```

Comment:
pre-existing: "library" here should be "module"

Reply:
Actually I don't think we need to conditionalize this at all. Why not just say "The *locator* argument should be a ```:class:`<file-locator>` ``` or a ```:drm:`<string>` ``` that can be coerced to one.